migrations/versions/c79589787f2f_remove_unused_columns.py

```diff
@@ -0,0 +1,41 @@
+"""remove unused columns from nicety and user tables
+
+Revision ID: c79589787f2f
+Revises: 67ac3b7d5c2f
+Create Date: 2022-01-20 18:36:18.328917
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "c79589787f2f"
+down_revision = "67ac3b7d5c2f"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_column("nicety", "starred")
+    op.drop_column("user", "autosave_enabled")
+    op.drop_column("user", "autosave_timeout")
+
+
+def downgrade():
+    op.add_column(
+        "nicety",
+        sa.Column("starred", sa.BOOLEAN(), autoincrement=False, nullable=True),
+    )
+    op.add_column(
+        "user",
+        sa.Column(
+            "autosave_enabled", sa.BOOLEAN(), autoincrement=False, nullable=True
+        ),
+    )
+    op.add_column(
+        "user",
+        sa.Column(
+            "autosave_timeout", sa.INTEGER(), autoincrement=False, nullable=True
```

Comment:
Do we need to explicitly set `autoincrement=False` on these columns? The [autoincrement docs](https://docs.sqlalchemy.org/en/13/core/metadata.html#sqlalchemy.schema.Column.params.autoincrement) say:

> Set up “auto increment” semantics for an integer primary key column. The default value is the string "auto" which indicates that a single-column primary key that is of an INTEGER type with no stated client-side or python-side defaults should receive auto increment semantics automatically; all other varieties of primary key columns will not.

Since this isn't a primary key, and the other columns are boolean, I think we can omit the argument and the default behavior will do what we want - right?

Reply:
You're right, no need to set this. Looking at the original migration I also see we never explicitly set it then either.